diesel/Cargo.toml

```diff
@@ -46,6 +46,7 @@ unstable = []
 lint = ["clippy"]
 large-tables = []
 huge-tables = ["large-tables"]
+128-column-tables = ["huge-tables"]
```

Comment:
FYI correct feature name would of course be `humongous-tables`

Reply:
massive-tables?

Reply:
Let's not start this again lol